common/source/docs/common-compassless.rst

```diff
@@ -0,0 +1,37 @@
+.. _common-compassless:
+
+===========================
+Operation without a Compass
+===========================
+
+Operation without a compass has been possible for Plane for a long time. However, recent additions of a compass/yaw source fallback mechanism in ArduPilot has opened the possibility of operations of  VTOL vehicles, such as Copters and QuadPlanes, without the use of a compass.
```

Comment:
I think "recent additions of a compass/yaw source fallback mechanism in ArduPilot" could be changed to "ArduPilot 4.1 (and higher) includes a Gaussian Sum Filter (GSF) which may allow Copter, Rovers and QuadPlanes to fly without a compass"